tardis_line_id.py

```diff
@@ -153,17 +153,23 @@ def plot_summary(
         else:
             self.lam_max = lam_max
 
-        if nlines == None:
-            self.nlines = 20
-        else:
-            self.nlines = nlines
-
         _lines_count = self.lines_count[np.argsort(self.lines_count)][::-1]
         _lines_fraction = self.lines_count[np.argsort(self.lines_count)][
             ::-1
         ] / float(self.lines_count.sum())
         _lines_ids = self.lines_ids_unique[np.argsort(self.lines_count)][::-1]
 
+        if nlines == None:
```

Comment:
The typical way to compare to `None` is as follows: `if nlines is None`

Reply:
Okay - I have changed this